test/gurka/gurka.h

```diff
@@ -818,6 +818,37 @@ void expect_maneuvers(const valhalla::Api& result,
       << "Actual maneuvers didn't match expected maneuvers";
 }
 
+/**
+ * Tests whether the expected set of instructions is emitted for the specified maneuver index.
+ * Looks at the output of Odin in the result.
+ *
+ * @param result the result of a /route or /match request
+ * @param maneuver_index the specified maneuver index to inspect
+ * @param expected_instructions the set of four instructions expected in the DirectionsLeg for the
+ *                              route at specified maneuver index.
+ *                              The four instructions shall be in this order:
+ *                                 text_instruction
+ *                                 verbal_transition_alert_instruction
+ *                                 verbal_pre_transition_instruction
+ *                                 verbal_post_transition_instruction
+ */
+void expect_instructions_at_maneuver_index(const valhalla::Api& result,
+                                           int maneuver_index,
+                                           const std::vector<std::string>& expected_instructions) {
+
+  EXPECT_EQ(result.directions().routes_size(), 1);
+  EXPECT_EQ(result.directions().routes(0).legs_size(), 1);
+  EXPECT_TRUE((maneuver_index >= 0) &&
```

Comment:
Nit: these should be `ASSERT_` since the upcoming `EXPECT`s aren't valid if these conditions aren't met.

Reply:
@danpaz thanks